acceptance/framework/connhelper/connect_helper.go

```diff
@@ -300,15 +307,17 @@ func (c *ConnectHelper) CreateResolverRedirect(t *testing.T) {
 }
 
 // TestConnectionFailureWithoutIntention ensures the connection to the static
-// server fails when no intentions are configured.
-func (c *ConnectHelper) TestConnectionFailureWithoutIntention(t *testing.T, clientType ...string) {
+// server fails when no intentions are configured. When provided with a ClientType option
+// the client is overridden, otherwise a default will be used
+func (c *ConnectHelper) TestConnectionFailureWithoutIntention(t *testing.T, connHelperOpts ConnHelperOpts) {
 	logger.Log(t, "checking that the connection is not successful because there's no intention")
 	opts := c.KubectlOptsForApp(t)
 	//Default to deploying static-client. If a client type is passed in (ex. job-client), use that instead.
 	client := StaticClientName
-	if len(clientType) > 0 {
-		client = clientType[0]
+	if connHelperOpts.ClientType != "" {
```

Comment:
**suggestion:** All the way through these I would named it connHelperOpts -> opts.

Reply:
Discussed with Curt, will leave it because opts is used in most of the functions for the KubectlOptions